ui/component/or-rules/src/or-rule-viewer.ts

```diff
@@ -20,7 +20,7 @@ import {translate} from "@openremote/or-translate";
 import {InputType, OrInputChangedEvent, OrMwcInput} from "@openremote/or-mwc-components/or-mwc-input";
 import i18next from "i18next";
 import {GenericAxiosResponse} from "@openremote/rest";
-import {showErrorDialog} from "@openremote/or-mwc-components/or-mwc-dialog";
+import {showErrorDialog, showOkDialog} from "@openremote/or-mwc-components/or-mwc-dialog";
```

Comment:
This import can also be removed now that the dialog is no longer used.

```suggestion
import {showErrorDialog} from "@openremote/or-mwc-components/or-mwc-dialog";
```